lib/sensors/accelerometer.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <unistd.h>
+
+#include <cstdint>
+#include <cstdio>
+#include <optional>
+
+#include "core/types.hpp"
```

Comment:
`<core/types.hpp>` i think, not entirely sure when `""` vs `<>` should be used

Reply:
use `<>` in general, unless it's local (i.e. relative to the current file)